tensorflow/tensorboard/components/tf-dashboard-common/dashboard-style.html

```diff
@@ -54,6 +54,14 @@
         display: block;
       }
 
+      .log-option-button {
```

Comment:
this CSS should actually be in tf-event-dashboard, not tf-dashboard-common, since it is only for the event dashboard.